Formula/cdktf.rb

```diff
@@ -16,8 +16,8 @@ class Cdktf < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "ba5b8e53e095c31190d725d5b1bfc8292d6f73da31da893bd257e6fc71747d45"
   end
 
+  depends_on "terraform" => :test
```

Comment:
Why?

Reply:
no need to install the latest `terraform`

Reply:
But you need terraform to use this tool. It is the literal definition of depending on terraform.

Reply:
yeah, but the latest, it would potentially suffer the same issue as terragrunt and etc.

Reply:
Some discussion thread in here, https://github.com/orgs/Homebrew/discussions/4037